engine/src/main/java/org/terasology/config/flexible/FlexibleConfigImpl.java

```diff
@@ -111,7 +111,7 @@ public void save(Writer writer) {
         for (Map.Entry<SimpleUri, Setting> entry : settings.entrySet()) {
```

Comment:
Please let's add to this method the functionality to save a human-readable description of the file content, so that by opening the file in notepad that's the first thing the user sees. 

That description should probably be stored as a read-only `description` field set on FC construction. The load method below should simply ignore the description it finds in a file.

Reply:
The only way I see to add the description to the JSON representation (@vampcat might be able to help out here) is to add a property named "description" or "_comment" or similar. Which one do you prefer? I prefer description, but if anyone has any other ideas I'd be glad to hear them.

Reply:
Sure, a "description" property at root level will do.